src/OpenDebugAD7/packages.config

```diff
@@ -2,7 +2,8 @@
 <packages>
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net45" />
   <package id="Microsoft.VisualStudio.Debugger.Interop.Portable" version="1.0.1" targetFramework="net45" />
-  <package id="MicroBuild.Core" version="0.2.0" targetFramework="net45" />
+  <package id="Microsoft.VisualStudio.Debugger.Interop.16.0" version="16.7.30328.139" targetFramework="net462" />
```

Comment:
I guess this still needs to be added to some packaging code.

Reply:
I'll take a look to see what needs to be fixed.